integration-tests/jpa-mariadb/pom.xml

```diff
@@ -198,15 +195,10 @@
                                         <!-- Speed things up a bit by not actually flushing writes to disk -->
                                         <tmpfs>/var/lib/mysql</tmpfs>
                                         <wait>
-                                            <!-- good docs found at: http://dmp.fabric8.io/#build-healthcheck -->
-                                            <tcp>
-                                                <mode>direct</mode>
-                                                <ports>
-                                                    <port>3306</port>
-                                                </ports>
-                                            </tcp>
-                                            <!-- Unfortunately booting MariaDB is slow, needs to set a generous timeout: -->
-                                            <time>40000</time>
+                                            <time>20000</time>
+                                            <exec>
+                                                <postStart>mysqladmin ping -h localhost -u hibernate_orm_test -phibernate_orm_test</postStart>
```

Comment:
So we assume that `mysqladmin` will need to be installed to run this test?

Reply:
jpa-mariadb uses the same approach as jpa-mysql

https://github.com/quarkusio/quarkus/blob/master/integration-tests/jpa-mysql/pom.xml#L201

Reply:
I see we already do that for `jpa-mysql`, so I guess it's OK :)

Reply:
I don't have mysqladmin on my path and it works for me.
`exec` command runs inside the container. 
looking at https://dmp.fabric8.io/#start-wait

Reply:
Oh, that's cool. Thanks for checking that out!